backend/strawberry_forms/mutations.py

```diff
@@ -101,9 +104,10 @@ def mutate(root, info, input: input_type) -> output:
             def mutate(root, info) -> output:
                 return _mutate(root, info, {})
 
-        mutate.__name__ = f"{name}Output"
-        mutate.name = name
-        mutate = strawberry.mutation(mutate, description=cls.__doc__)
+        # Hack because something changed and now the name contains `Mutation`
+        # as the classname, which makes sense but why it didn't happen before? lol
+        name = name.replace("Mutation", "")
```

Comment:
so the name of the class is `ResetPasswordMutation`, assigned to `reset_password = ResetPasswordMutation`

Before the update of strawberry the mutation name used to be `resetPassword` but now it is `resetPasswordMutation`

This is a quick hack as I plan on removing the django forms integration anyway :)